policy/modules/contrib/networkmanager.te

```diff
@@ -526,6 +530,27 @@ optional_policy(`
 	dbus_system_domain(NetworkManagerDispatcher_t, NetworkManagerDispatcher_exec_t)
 ')
 
+########################################
+#
+# NetworkManager-sudo local policy
+#
+
+allow NetworkManagerSudo_t self:capability dac_override;
```

Comment:
Just a note this needs to be clarified with nm developers.

Reply:
Sure. I'll write them.